src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -209,11 +213,33 @@ public void joystickDrive(double inputX, double inputY, double inputRotation) {
 
 	public void drive(ChassisSpeeds chassisSpeeds) {
 		SwerveModuleState[] states = m_kinematics.toSwerveModuleStates(chassisSpeeds);
+        //if in lock mode, manually sets speed and angles of swerve modules
+		if (m_lockMode) {
+			states[0].speedMetersPerSecond = -0.01;
+			states[1].speedMetersPerSecond = -0.01;
+			states[2].speedMetersPerSecond = 0.01;
+			states[3].speedMetersPerSecond = 0.01;
```

Comment:
I don't think we want to set the speeds other than 0. I know the other team did, but I don't think we need to strain the motors for this.